Data/Binary/Serialise/CBOR/Term.hs

```diff
@@ -102,7 +102,8 @@ decodeTerm :: Decoder Term
 decodeTerm = do
     tkty <- peekTokenType
     case tkty of
-      TypeUInt   -> fromWord <$> decodeWord
+      TypeUInt   -> do w <- decodeWord
+                       return $! fromWord w
```

Comment:
True, I suppose we could conditionally define `<$!>` locally (we need to support GHC back to 7.8 at least). I don't think there's any need to force the `a` if `g` is strict, which it usually is, but I guess it makes a small difference in the lazy case.